.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.3.1
   - 2.4.5
```

Comment:
Can you also add the latest 2.5 and 2.6 here?  Then we can be sure it works with newer rubies.